### PR TITLE
[4] correct param that is obviously wrong

### PR DESCRIPTION
**Pull request by PhilETaylor**
code review

### CODE REVIEW

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on a025d8571f5ce9afcc50f1f78e517008329c1bb5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33950">issues.joomla.org/tracker/joomla-cms/33950</a>.</sub>

File: administrator/components/com_contenthistory/src/Helper/ContenthistoryHelper.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public static function createObjectArray($object)
 	/**
 	 * Method to decode JSON-encoded fields in a standard object. Used to unpack JSON strings in the content history data column.
 	 *
-	 * @param   \stdClass  $jsonString  Standard class object that may contain one or more JSON-encoded fields.
+	 * @param   string  $jsonString  JSON String to covert to an object.
```
Comment (ReLater) [inline administrator/components/com_contenthistory/src/Helper/ContenthistoryHelper.php]:
```suggestion
	 * @param   string  $jsonString  JSON String to convert to an object.
```

**Comment by richard67** [comment]
I've restored the previous test result since the commit after that was just a change in a comment.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 5ba6892586317d652ed4e5b13efa85e30199c862<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33950">issues.joomla.org/tracker/joomla-cms/33950</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33950">issues.joomla.org/tracker/joomla-cms/33950</a>.</sub>

**Comment by wilsonge** [comment]
Thanks. Although this looks like one of the weirdest functions in human history....

**Pull request merged**